spec/parser_spec.lua

```diff
@@ -419,6 +419,19 @@ describe("Parser /", function()
             ]], "local function name has a ':'")
         end)
 
+        it("disallow too many function parameters", function ()
```

Comment:
This looks like it has an indentation of only 3 spaces?

Reply:
Ah! I tweaked my `vimrc` and ended up messing with something that may have led to this😅

Reply:
While you're checking the vimrc, it might be a good idea to verify if the editorconfig plugin is working.

By the way, this 3-spaces indentation reminds me of Hisham. He's not afraid to swim against the current, and argued that 3 spaces is good for Lua because it aligns the "end" tokens.

Reply:
Good point, I totally forgot to setup editorconfig on the WSL.
I prefer tabs personally because I can render them at any width, so I do 2 space-wide tabs for Lua generally, but 
looking at it like that 3-space tabs sounds like a good argument.